static/js/muting.js

```diff
@@ -4,14 +4,15 @@ var exports = {};
 
 var muted_topics = new Dict({fold_case: true});
 
-exports.mute_topic = function (stream, topic) {
+exports.mute_topic = function (stream, topic, manual_trigger) {
     var sub_dict = muted_topics.get(stream);
     if (!sub_dict) {
         sub_dict = new Dict({fold_case: true});
         muted_topics.set(stream, sub_dict);
     }
     sub_dict.set(topic, true);
     unread.update_unread_counts();
+    muting_ui.mute_notification(stream, topic, manual_trigger);
```

Comment:
Can we let either ui.js or muting_ui.js orchestrate the notification?  I am hoping to keep muting.js mostly free of UI dependencies.